components/footer.tsx

```diff
@@ -1,3 +1,50 @@
-export default function Footer() {
-  return <footer className="bg-light px-4 py-8 md:p-16">footer</footer>
+/* eslint-disable @next/next/no-img-element */
+import { Button } from './data-components/button'
+import FooterMenu from './footer-menu'
+import { PageType } from './types'
+import ContactData from '@/components/data-components/contact-data'
+
+export default function Footer({ data }: PageType) {
+  return (
+    <div className="hidden md:block ">
+      <footer className="bg-light p-16 grid grid-cols-4 ">
+        <img src="/icons/veahavta-icon.svg" alt="ואהבת לוגו" />
+        <div className="grid justify-self-center self-center">
+          <Button
+            className={
+              'min-w-[130px] text-base text-header-blue px-8 py-2 rounded-full mb-1 border-2 border-solid font-bold'
+            }
+          >
+            {data.common.footerScheduleButton}
+          </Button>
+          <Button
+            className={
+              'min-w-[130px] text-base text-header-blue px-8 py-2 rounded-full mt-1 border-2 border-solid font-bold'
+            }
+          >
+            {data.common.footerSupportUsButton}
+          </Button>
+        </div>
+        <FooterMenu data={data}></FooterMenu>
+        <div className="self-center">
+          <h2 className="font-bold text-2xl">
+            {data.homepage.contactUsHeading}
+          </h2>
+          <div className="text-sm">
+            {data.common.contactUsLinks.map((contactUsLinks: any, key: any) => {
```

Comment:
We could define a type ContactUsLink instead of any, also in the Data type,
You could extract the type from:
```typescript
export type Common = {
  contactUsLinks: Array<{
    text: string
    linkType: string
    imagePath: string
  }>
  ...
}
```

So it would become:
```typescript
export type ContactUsLink = {
    text: string
    linkType: string
    imagePath: string
}
  
export type Common = {
  contactUsLinks: Array<ContactUsLink>
  ...
}
```